setup.py

```diff
@@ -64,7 +64,8 @@
     'Topic :: Security',
     'Topic :: Software Development'
   ],
-  packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
+  packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests",
+      "debian"]),
```

Comment:
Unrelated: 
Funny thing, we are not excluding the only `test` directory of the project. :)
For our setup the following line would be what we want:
```python
packages=find_packages(exclude=["test", "test.*", "debian"])
```
Or maybe explicitly including packages would be even easier?,
```python
packages=["in_toto", "in_toto.gpg", "in_toto.models"]
```

Worth an extra PR?

Reply:
Fixed with https://github.com/in-toto/in-toto/pull/173. Please merge there first and resolve conflict here.